proto/tendermint/abci/types.proto

```diff
@@ -366,10 +366,8 @@ message ResponseProcessProposal {
   // Changes to consensus-critical gas, size, and other parameters.
   tendermint.types.ConsensusParams consensus_param_updates = 5;
 
-  // Core chain lock that will be used for generated block.
-  tendermint.types.CoreChainLock core_chain_lock_update = 100;
```

Comment:
Ivan and I agreed that leaving this field does not make sense, since this value may differ due to the asynchronous work core node. In other words, only the proposer can propose this value, the rest members can only verify it.
lets discuss this with @shumkov